script/tool/lib/src/publish_plugin_command.dart

```diff
@@ -120,7 +211,50 @@ class PublishPluginCommand extends PluginCommand {
           remoteUrl: remoteUrl,
           shouldPushTag: shouldPushTag);
     }
-    await _finishSuccesfully();
+    _print('Release ${packageDir.basename} successful.');
```

Comment:
There aren't any bools for the sub-steps; does that mean they are fatal errors when something goes wrong?

If we try to release 3 packages, an error with one of them presumably shouldn't kill the whole thing, but instead move on to the next, and then at the end list all the successes and failures.